src/ChildProcessUtilities.js

```diff
@@ -74,6 +62,35 @@ export default class ChildProcessUtilities {
     }
   }
 
+  static spawnStreaming(command, args, opts, prefix, callback) {
+    opts = Object.assign({}, opts, {
+      stdio: ["ignore", "pipe", "pipe"],
```

Comment:
Hi. Sorry to bother you on such an old PR but I am curious about the motivation for ignoring stdin? I understand that it could not really make sense in some case (that I don't have in mind - I am not an stdin/out professional), but I am facing [an issue](https://github.com/lerna/lerna/issues/1745) and switching `ignore` with `pipe` or `inherit` seems to change nothing in most case but could help me with my issue.

Reply:
I think the biggest problem with `"inherit"` is in concurrent execution: which process "gets" to have `stdin`, should it be necessary? Imagine 8 concurrent `npm` executions demanding a one-time password, for example.

I _am_ sympathetic to this, but I think we would need an option, like `--inherit-stdin`, to explicitly enable this in both `lerna exec` and `lerna run`. Open questions:

* Should `--inherit-stdin` imply `--stream`?
* Should `--inherit-stdin` be allowed if `--parallel` is enabled?

There are two options, as I see it, to avoid the interleaved/conflicting `stdin` problem:

1. `--inherit-stdin` implicitly restricts concurrency to `1`
2. Recommend `--concurrency 1` in docs, warn loudly when `this.concurrency !== 1`